lib/config.ts

```diff
@@ -30,6 +31,15 @@ export function validateConfig(params: ConfigParams<any>): ConfigInterface {
   if (!('isCustomStoreApp' in params) || !params.isCustomStoreApp) {
     mandatory.push('scopes');
   }
+  // ENABLE THIS CHECK AS PART OF THE NEXT MAJOR RELEASE
+  // if ('isCustomStoreApp' in params && params.isCustomStoreApp) {
```

Comment:
We could add an actual check here for the version, right? With this and the optional test, I'm starting to wonder if it wouldn't be nice to have a method like `deprecated` that only runs code if VERSION >= something. That would make it easy to search-replace when we're releasing a new version.